packages/linter/src/builders/lint/schema.json

```diff
@@ -70,6 +71,15 @@
       "type": "boolean",
       "description": "Fixes linting errors (may overwrite linted files).",
       "default": false
+    },
+    "cache": {
+      "type": "boolean",
+      "description": "Only check changed files.",
+      "default": false
+    },
+    "outputFile": {
+      "type": "string",
+      "description": "Output file for formatted output."
```

Comment:
This could probably be a bit clearer that it's the lint report. Maybe copy the text eslint uses - `Specify file to write report to`